### PR TITLE
add device MAX6675

### PR DESCRIPTION
**Pull request by MaBecker**
Note: This is the previous one to module MAX31855

### CODE REVIEW

File: devices/MAX6675.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Usage is very simple. Using the example pins shown above:
+
+```JavaScript 
+MISO = B4; SCK = B3; CS = C0;    // Espruino 
+//MISO = A6; SCK = A5; CS = A7;    // PICO or Espruino WiFi
+//MISO = NodeMCU.D6; SCK = NodeMCU.D5; CS = NodeMCU.D7; // NodeMCU
+//MISO = D12 ;SCK = D14; CS = D13; // ESP8266 >= ESP12
+
+SPI1.setup({ miso:MISO, sck:SCK, baud:1000000 });  
+
```
Comment (gfwilliams) [inline devices/MAX6675.md]: Are you missing a `var max = require("MAX6675").connect...` here?

**Comment by gfwilliams** [comment]
Thanks! Looks really good - just made a note about the docs though. Is that right?

File: devices/MAX6675.md (hunk 1 above)
Comment (MaBecker) [reply devices/MAX6675.md]: Thanks for this finding, will send updates.

**Comment by MaBecker** [comment]
That's it :)

**Comment by gfwilliams** [comment]
Thanks!

**Pull request merged**